functions-and-scope.js

```diff
@@ -70,6 +107,15 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan opslaan?
 // Log het antwoord in de terminal.
 
+let highest = grades[0];
```

Comment:
Super dat je met `grades[0]` bent begonnen in plaats van 0. Het is een veelgemaakte fout, dus lekker bezig! 🏅